common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -1667,7 +1667,7 @@ public static PrefCalendarInitialView fromString(String s) throws ServiceExcepti
     public static enum PrefClientType {
         standard("standard"),
         advanced("advanced"),
-        zimbrax("zimbrax");
```

Comment:
Removing 'zimbrax' won't have an affect on exiting deployments, will it?

Reply:
i don't think so, since this change is for z9. It was changed in zimbra-attr.xml as part of PREAPPS-4182 but i think they missed to do `generate-getters`.